app/controllers/admin/collections_controller.rb

```diff
@@ -217,4 +217,40 @@ def destroy
       redirect_to admin_collection_path(@source_collection)
     end
   end
+
+  # POST /collections/1/batch_ingest
+  def batch_ingest
+    if @collection.dropbox.manifest_processable?
+      # Spawn off background job
+      AvalonJobs.collection_batch_ingest(@collection.id)
+      flash[:notice] = t('batch_ingest.initiated', name: @collection.name)
+    else
+      flash[:error] =
+        if @collection.dropbox.manifest_error?
+          t('batch_ingest.error', name: @collection.name)
+        elsif @collection.dropbox.manifest_processing?
+          t('batch_ingest.processing', name: @collection.name)
+        elsif @collection.dropbox.manifest_processed?
+          t('batch_ingest.processed', name: @collection.name)
+        else
+          t('batch_ingest.not_found', name: @collection.name)
+        end
+    end
+    redirect_to admin_collection_path(@collection)
+  end
+
+  # GET /collections/1/batch_log
+  def batch_log
+    # View most recent batch log
+    # TODO ...
+    text_output =
+      Avalon::Batch::Manifest.concatenate_log_files(@collection.dropbox.base_directory)
+
+    unless text_output.nil?
```

Comment:
Style comment. Should never use `unless` with an `else`. Always use the positive case first. Much easier to read this way. 

Rubocop guideline here: https://github.com/bbatsov/ruby-style-guide#no-else-with-unless

So easy change is just do `if text_output.present?` instead